implementation/src/main/java/io/smallrye/metrics/MetricsRegistryImpl.java

```diff
@@ -49,10 +54,26 @@
 public class MetricsRegistryImpl extends MetricRegistry {
 
     private static final Logger log = Logger.getLogger(MetricsRegistryImpl.class);
+    private static final String NONE = "NONE";
 
     private Map<String, Metadata> metadataMap = new HashMap<>();
     private Map<String, Metric> metricMap = new ConcurrentHashMap<>();
 
+    Optional<String> globalTags;
```

Comment:
This could be `final` (if we change the constructor a little bit to only assign to it once)